tests/components/minio/test_minio.py

```diff
@@ -185,6 +199,5 @@ def event_callback(event):
     }
 
     assert DOMAIN == call_domain
-    assert json.dumps(expected_event, sort_keys=True) == json.dumps(
-        call_event, sort_keys=True
-    )
+    assert json.dumps(expected_event,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)